.github/workflows/tag-and-build.yml

```diff
@@ -84,6 +84,13 @@ jobs:
       run: make install-operator-sdk
 
     - name: Adjust Compatibility Matrix in readme
+      run: |
+        sed -i -E "s/(.*CodeFlare Operator.*)v[0-9]+\.[0-9]+\.[0-9]+(.*\|.*\|)/\1${{ github.event.inputs.version }}\2/" README.md
```

Comment:
There is no need for the regex alteration.
If you call the regex below twice it will replace both values.
Not sure if there is a possibility to do both replacements in one command.

Reply:
```sed -i -E "s/(.*CodeFlare Operator.*)v[0-9]+\.[0-9]+\.[0-9]+(.*)v[0-9]+\.[0-9]+\.[0-9]+(.*)/\1${{ github.event.inputs.version }}\2${{ github.event.inputs.version }}\3/" README.md```

Single line sed, I tested this in my local repo and it seems to work well.